docs/upgrade.md

```diff
@@ -11,23 +11,24 @@ Check the current version of SC4S by running ```sudo <docker or podman> logs SC4
 
 ```
 [Service]
-Environment="SC4S_IMAGE=ghcr.io/splunk/splunk-connect-for-syslog/container2:2"
+Environment="SC4S_IMAGE=ghcr.io/splunk/splunk-connect-for-syslog/container3:latest"
 ```
 
 Restart the service
 ```sudo systemctl restart sc4s```
 
-Using the "1" version is recommended, but a specific version can be specified in the unit file if desired:
+Using the "3" version is recommended, but a specific version can be set in the unit file if desired:
 
 ```
 [Service]
-Environment="SC4S_IMAGE=ghcr.io/splunk/splunk-connect-for-syslog/container2:2.91.0"
+Environment="SC4S_IMAGE=ghcr.io/splunk/splunk-connect-for-syslog/container3:2.49.0"
```

Comment:
should nt be latest ?

Reply:
This is an example of choosing specific version, though it should be 3.0.0 for container3 because there is no other version yet.